x-pack/legacy/plugins/canvas/canvas_plugin_src/uis/views/image.js

```diff
@@ -22,14 +23,14 @@ export const image = () => ({
     },
     {
       name: 'mode',
-      displayName: 'Fill mode',
-      help: 'Note: Stretched fill may not work with vector images',
+      displayName: ViewStrings.Image.args.mode.getDisplayName(),
+      help: ViewStrings.Image.args.mode.getHelp(),
```

Comment:
Same here... this is really lengthy.  I'd really prefer we keep the dictionaries shallow.